requirements.txt

```diff
@@ -1,4 +1,32 @@
+cachetools==4.2.1
+certifi==2020.12.5
+chardet==4.0.0
 click==7.1.2
+geojson==2.5.0
+google-api-core==1.26.1
+google-api-python-client==2.0.2
+google-auth==1.28.0
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.3
+googleapis-common-protos==1.53.0
+httplib2==0.19.0
+idna==2.10
+oauthlib==3.1.0
+packaging==20.9
+protobuf==3.15.6
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
 PyGithub==1.52
+PyJWT==2.0.1
 pyowm==3.0.0
+pyparsing==2.4.7
+PySocks==1.7.1
 python-coinmarketcap==0.2
+pytz==2021.1
+requests==2.25.1
+requests-oauthlib==1.3.0
+rsa==4.7.2
+six==1.15.0
+uritemplate==3.0.1
+urllib3==1.26.3
+wrapt==1.12.1
```

Comment:
wow - a lot of packages - are these all needed for the Calendar API?

Reply:
I don't really want to include all these packages if we can; makes the package pretty heavy for one command

Reply:
Let me double check, but I just followed the googles api guide and install whatever they've suggested.

Reply:
Yeah, it is all dependancies that google calendar api introduces.
At least it is the list that I got when I did `pip freeze > requirements.txt`. Is there a better way to handle dependancies?

In general, if it is too heavy, I think we can close this PR, not a critical feature :)

Reply:
Using `pip freeze` is fine but it is listing out all the installed packages dependencies also. I would prefer a smaller `requirements.txt` which just the core packages and then allowing `pip` to handle the dependencies versions rather than explicitly stating all of them. I think it allows for clearer commits when adding commands.

Again thought, I'm not going to be picky as you've put lots of effect into this! 😄 👍